website/scripts/get-average-bug-and-pr-age.js

```diff
@@ -0,0 +1,168 @@
+module.exports = {
+
+
+  friendlyName: 'Deliver Fleet GitHub repo statistics',
+
+
+  description: 'Get the average age of open pull requests and issues with the bug label in the fleetdm/fleet GitHub repo.',
+
+
+  exits: {
+
+
+  },
+
+
+  fn: async function () {
+
+    sails.log('Getting average open time for issues with the "bug" label and open pull requests in the fleetdm/fleet Github repo...');
+
+
+    let baseHeaders = {
+      'User-Agent': 'Fleet average open time',
+    };
+
+    const oneDayInMs = (1000 * 60 * 60 * 24);
```

Comment:
nit: consider writing this in all caps: `ONE_DAY_IN_MS` (to make it clear it's a constant anywhere it is used)